### ISSUE TITLE
[Question] DDC Notebook doesn't have DDC trained TTS

### ISSUE THREAD

**Issue by george-roussos** [mozilla/TTS#529]
Hi, so the notebook here https://colab.research.google.com/drive/1u_16ZzHjKYFn1HNVuA4Qf_i2MMFB9olY?usp=sharing&authuser=1#scrollTo=FuWxZ9Ey5Puj says Taco2 is trained using DDC, but in the TTS config, DDC is actually disabled and prenet is dropout and not bn. Does it mention another model?

**Comment by erogol** [comment]
You do not use DDC at inference time. It is trained with DDC but it only uses the fine-grain decoder at inference. 

And yes this model uses normal Prenet since I forgot to change it to BN.

**Comment by george-roussos** [comment]
Oh thanks. I thought it was trained and not used during inference, but in my TTS (which I also trained using DDC), if I disable DDC in the config.json during inference, it doesn't work. I get `Unexpected key(s) in state_dict: "coarse_decoder.prenet.linear_layers.0.linear_layer.weight", "coarse_decoder.prenet.linear_layers.1.linear_layer.weight", "coarse_decoder.attention_rnn.weight_ih", "coarse_decoder.attention_rnn.weight_hh", "coarse_decoder.attention_rnn.bias_ih", "coarse_decoder.attention_rnn.bias_hh", "coarse_decoder.attention.query_layer.linear_layer.weight", "coarse_decoder.attention.inputs_layer.linear_layer.weight", "coarse_decoder.attention.v.linear_layer.weight", "coarse_decoder.attention.v.linear_layer.bias", "coarse_decoder.attention.ta.weight", "coarse_decoder.attention.ta.bias", "coarse_decoder.attention.location_layer.location_conv1d.weight", "coarse_decoder.attention.location_layer.location_dense.linear_layer.weight", "coarse_decoder.decoder_rnn.weight_ih", "coarse_decoder.decoder_rnn.weight_hh", "coarse_decoder.decoder_rnn.bias_ih", "coarse_decoder.decoder_rnn.bias_hh", "coarse_decoder.linear_projection.linear_layer.weight", "coarse_decoder.linear_projection.linear_layer.bias", "coarse_decoder.stopnet.1.linear_layer.weight", "coarse_decoder.stopnet.1.linear_layer.bias".` Am I doing something wrong? I train using DDC 7 and then during inference I disable it in config.

**Comment by erogol** [comment]
you need to enable DDC. To reduce the model size I removed DDC layers for the colab example.
